app/controllers/search_controller.rb

```diff
@@ -24,7 +29,9 @@ def index
     if search_params.no_search? && params[:format] != "json"
       render(action: 'no_search_term') && return
     end
-    search_response = SearchAPI.new(search_params).search
+
+    variant = search_ab_test_variant.variant_name
+    search_response = SearchAPI.new(search_params, format_boosting: variant).search
```

Comment:
we have called the variant `format_boosting` and the test `FormatWeighting` can we get these names to match?